collectors/bzimport/tests/test_fixups.py

```diff
@@ -24,7 +24,7 @@ def test_affect_ps_module_fixup(self):
 
 class TestFlawFixer:
     @pytest.mark.parametrize(
-        "summary,title,component,components",
+        "cve_description,title,component,components",
```

Comment:
This should also remain the same as this is supposed to be summary coming from Bugzilla. This applies for the whole test below.